projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -27,6 +27,10 @@
         cursor: default;
     }
     /* stylelint-enable */
+
+    //&[data-size='l'] {
```

Comment:
hi! what styles should I add here?

Reply:
Size L is fine, for Size M width of country picker should be 5.5rem, for Size S is harder, it would probably require some hacks because minimum width of CustomContent is 2rem which does not allow to shrink it any further. It should also align flag in the select with flags in dropdown. You would need to play with styles a bit.

Reply:
<img width="584" alt="Screenshot 2022-12-28 at 19 47 21" src="https://user-images.githubusercontent.com/22116465/209859354-126f2513-1aa6-4c98-9ac7-51830beb5677.png">

Reply:
Looks good to me 👍